backstop/tests/side-navigation.html

```diff
@@ -67,165 +67,195 @@ <h1>Side navigation</h1>
+                  aria-hidden="true"
+                >
+                  <path
+                    d="m14.9 3.6 1.1-2.9-.7-.7-2.8 1.1-.5 1.5-5.6 5.6a1.65413 1.65413 0 0 0 -2 .3l-3.9 3.9a1.63393 1.63393 0 0 0 0 2.4l.7.7a1.63393 1.63393 0 0 0 2.4 0l3.9-3.9a1.84256 1.84256 0 0 0 .3-2l5.6-5.6m-9.9 3.9.2-.2a2.93516 2.93516 0 0 1 1.9-.7 1.26834 1.26834 0 0 1 .6.1l1.6-1.6a4.09721 4.09721 0 0 0 -2.2-5.2 4.29961 4.29961 0 0 0 -2.5-.2l2.9 2.9-3 3-2.9-2.9a4.00641 4.00641 0 0 0 2.9 4.8zm7 .4-1.6 1.6a2.85417 2.85417 0 0 1 0 1.3l4.5 4.5a1.39083 1.39083 0 0 0 1.9 0l.4-.4a1.39083 1.39083 0 0 0 0-1.9"
+                  />
+                </svg>
+                <span class="iui-label">Administration</span>
+              </button>
+              <span
+                class="iui-tooltip"
+                style="margin-bottom: 10px;"
+              >Administration</span>
+            </div>
           </div>
 
-          <div class="iui-bottom">
-            <button
-              class="iui-button iui-large iui-sidenav-button"
-              role="tab"
-              aria-selected="false"
-              aria-label="Administration"
-              aria-controls="administration-tab"
-              id="test-tab-2"
+          <!-- Expand arrow -->
+          <button
+            class="iui-button iui-expand iui-sidenav-button"
+            role="button"
+            aria-label="Expand menu"
+            onclick="handleHidePanel()"
+            id="test-expand"
+          >
+            <!-- Use chevron-right.svg from @itwin/itwinui-icons package. -->
+            <svg
+              class="iui-icon"
+              aria-hidden="true"
+              viewBox="0 0 16 16"
             >
-              <!-- Use setup.svg from @itwin/itwinui-icons package. -->
-              <svg
-                class="iui-icon"
-                viewBox="0 0 16 16"
-                aria-hidden="true"
-              >
-                <path
-                  d="m14.9 3.6 1.1-2.9-.7-.7-2.8 1.1-.5 1.5-5.6 5.6a1.65413 1.65413 0 0 0 -2 .3l-3.9 3.9a1.63393 1.63393 0 0 0 0 2.4l.7.7a1.63393 1.63393 0 0 0 2.4 0l3.9-3.9a1.84256 1.84256 0 0 0 .3-2l5.6-5.6m-9.9 3.9.2-.2a2.93516 2.93516 0 0 1 1.9-.7 1.26834 1.26834 0 0 1 .6.1l1.6-1.6a4.09721 4.09721 0 0 0 -2.2-5.2 4.29961 4.29961 0 0 0 -2.5-.2l2.9 2.9-3 3-2.9-2.9a4.00641 4.00641 0 0 0 2.9 4.8zm7 .4-1.6 1.6a2.85417 2.85417 0 0 1 0 1.3l4.5 4.5a1.39083 1.39083 0 0 0 1.9 0l.4-.4a1.39083 1.39083 0 0 0 0-1.9"
-                />
-              </svg>
-              <span class="iui-label">Administration</span>
-            </button>
-            <span
-              class="iui-tooltip"
-              style="margin-bottom: 10px;"
-            >Administration</span>
+              <path d="m4.7 0-1.4 1.4 6.6 6.6-6.6 6.6 1.4 1.4 8-8z" />
+            </svg>
+          </button>
+        </div>
+
+        <div class="iui-side-navigation-submenu">
+          <div class="iui-resizer">
+            <div class="iui-resizer-bar"></div>
```

Comment:
The resizer bar is conflicting with the vertical scroll bar.  I was thinking 1 solution might be to wrap the submenu content and have a 8px left/right margin so that the scrollbar appears 8px indented from the edge of the submenu.
![conflict](https://user-images.githubusercontent.com/849817/130642396-73d15e71-3935-42e6-8776-3c3545fea958.gif)

Reply:
Is it mandatory to have resizer now?

Reply:
Maybe we could deprioritize resizer and leave it as a CSS implementation using [`resize: horizontal`](https://developer.mozilla.org/docs/Web/CSS/resize)?

![image](https://user-images.githubusercontent.com/9084735/130827161-4b083ffd-9622-4069-9b61-e5443d44179e.png) 

The resize handle shows below the scrollbar so at least that's nice?
![image](https://user-images.githubusercontent.com/9084735/130827235-42c97af1-9da1-4cc2-bef1-b36086c43c0f.png)